src/rpc/net.cpp

```diff
@@ -272,6 +272,49 @@ static RPCHelpMan getpeerinfo()
     };
 }
 
+static RPCHelpMan addwhitelist()
```

Comment:
A question rather than a suggestion: should the docs highlight that these permissions do not persist beyond a node reboot? Intuitively, I'd have expected this kind of effect to be persistent, so maybe highlighting that is useful if I'm not the only one?

Reply:
It's intuitive for me these permissions aren't persistent since there is no way to remove them (thinking about a follow-up adding an option for removal in this RPC), but sounds goods for me to add it in the docs to clarify.